### PR TITLE
TrafficManager Endpoint management cmdlets

### PR DESCRIPTION
**Pull request by vladca**
This supersedes Alberto's PR from dev in order to include fresh Session Records but to be honest the previous Session Records looked fine. Checking into release-0.9.5 to catch the release train

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vladca__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vladca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by stankovski** [comment]
@azuresdkci add to whitelist

File: src/ResourceManager/Automation/Commands.Automation.Test/Commands.ResourceManagement.Automation.Test.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <ProductVersion>
     </ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{59D1B5DC-9175-43EC-90C6-CBA601B3565F}</ProjectGuid>
+    <ProjectGuid>{127D0D51-FDEA-4E1A-8CD8-34DEB5C2F7F6}</ProjectGuid>
```
Comment (markcowl) [inline src/ResourceManager/Automation/Commands.Automation.Test/Commands.ResourceManagement.Automation.Test.csproj]: Please remove any changes to this file

**Comment by markcowl** [comment]
On demand run kicked off here:  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-on-demand/25/

**Comment by markcowl** [comment]
Installer build kicked off here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-msi/346/

File: src/ResourceManager/Automation/Commands.Automation.Test/Commands.ResourceManagement.Automation.Test.csproj (hunk 1 above)
Comment (vladca) [inline src/ResourceManager/Automation/Commands.Automation.Test/Commands.ResourceManagement.Automation.Test.csproj]: That would be the wrong thing to do. this is another Project GUID mix-up between different solutions. I'll submit the proper fix instead - in ResourceManager.sln

**Comment by vladca** [comment]
All passing, please merge. Thanks!

**Pull request merged**